### PR TITLE
Recaptcha and geolocation token

### PR DESCRIPTION
**Pull request by ElfoLiNk**

### CODE REVIEW

**Review by AleDore — APPROVED** [review]
LGTM

File: code/variables.tf [hunk 1]
```diff
@@ -424,3 +424,9 @@ variable "io_apim_productid" {
   type    = string
   default = null
 }
+
+# Recaptcha
+variable "recaptcha_secret_key" {
```
Comment (uolter) [inline code/variables.tf:429]: @AleDore who is going to provide me this secret key? Should it be different from production and uat?

Comment (AleDore) [reply code/variables.tf:429]: Nope, we can use the same used in io-pay

File: code/api.tf [hunk 2]
```diff
@@ -126,6 +132,11 @@ data "azurerm_key_vault_secret" "backend_client_secret" {
   key_vault_id = azurerm_key_vault.key_vault.id
 }
 
+data "azurerm_key_vault_secret" "backend_geolocation_token" {
+  name         = "backend-GEOLOCATION-TOKEN"
```
Comment (uolter) [inline code/api.tf:136]: @AleDore ..... one more .... where do I get the value to put in this secret?
What is it the GEOLOCATION-TOKEN?

Comment (AleDore) [reply code/api.tf:136]: @uolter I will provide it later :)

**Review by uolter — APPROVED** [review]

**Pull request merged**